_search-plugins/search-shard-routing.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: default
+title: Search shard routing
+nav_order: 210
+---
+
+# Search shard routing
+
+To ensure redundancy and improve search performance, OpenSearch replicates index data across various shards located on different nodes. When a search query is executed, OpenSearch routes the request to a node containing either primary or replica shard of the index. This technique is known as _search shard routing_.
```

Comment:
this statement looks a bit inaccurate as its trying to conflate the purpose of both primary and replica shards whereas says `OpenSearch replicates`.  how about - 
To ensure redundancy and improve search performance, OpenSearch distributes index data across multiple primary shards, with each primary shard having one or more replica shards.